components/cdc/src/endpoint.rs

```diff
@@ -1132,13 +1149,16 @@ impl Initializer {
             }
             debug!("cdc scan entries"; "len" => entries.len(), "region_id" => region_id);
             fail_point!("before_schedule_incremental_scan");
-            let scanned = Task::IncrementalScan {
-                region_id,
-                downstream_id,
-                entries,
-            };
-            if let Err(e) = self.sched.schedule(scanned) {
-                error!("cdc schedule cdc task failed"; "error" => ?e, "region_id" => region_id);
+            if let Err(e) = self.sink_scan_events(entries, done, require_barrier).await {
```

Comment:
If we send the scan event here, then ticdc could still receive scanned entries after a region had been deregistered. Is that OK?

Reply:
@amyangfei PTAL